sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -110,9 +114,16 @@ object AkkaGrpcPlugin extends AutoPlugin {
             toGenerator(g, ScalaBinaryVersion(scalaBinaryVersion.value), generatorLogger))
         },
         // configure the proto gen automatically by adding our codegen:
-        // FIXME: actually specifying separate Compile and Test target stub and languages does not work #194
-        PB.targets ++=
-          targetsFor(sourceManaged.value, akkaGrpcCodeGeneratorSettings.value, akkaGrpcGenerators.value),
+        PB.targets := {
+          val configTargets =
+            targetsFor(sourceManaged.value, akkaGrpcCodeGeneratorSettings.value, akkaGrpcGenerators.value)
+          // To play nice with other sbt-protoc plugins that might have injected their own targets, don't simply
+          // override the key. Append instead, after excluding targets added by parent configurations (transitively)
+          // since they already ran and would cause protoc to freeze as their name collides with the own generators
```

Comment:
If

Reply:
https://github.com/thesamet/sbt-protoc/issues/108

Reply:
https://github.com/thesamet/sbt-protoc/pull/111 was merged, so I'll replace https://github.com/akka/akka-grpc/pull/702/commits/9b35ce64fc0ecfa8abfabcbf409577fe2387f6ab by a `sbt-protoc` bump once a new release is cut.

Reply:
Initial implementation replaced by the bump in https://github.com/akka/akka-grpc/pull/702/commits/76eba3c76e917c9396fa10f5f10ee3709313d0e6